gemfiles/rails_5_0.gemfile

```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rails", :git => "git@github.com:rails/rails.git"
```

Comment:
I know, that is what I am suggesting as a possible solution.  The error is an ssh error, so cloning the repo over https should avoid that problem in travis.